### PR TITLE
PestPHP documentation search extension

### PR DESCRIPTION
**Pull request by danyelkeddah**
## Description

PestPHP Documentation Extension will allow developers to access [PestPHP](https://pestphp.com/docs) documentation from Raycast

## Type of change

- New extension

## Screencast
raycast-pestphp-documentation-extension
raycast-pestphp-documentation-algolia-extension

## Checklist

- [X] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [X] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)

### CODE REVIEW

File: extensions/pestphp-documentation/src/pestphp.tsx [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+    url: string;
+  }[];
+};
+
+const documentation: { [key: string]: docList } = {
+  "v1.x": require("./documentation/v1.x.json"),
+};
+
+const APPID = "BH4D9OD16A";
+const APIKEY = "d25b0b281fd72e7b8ce770857fff45bf";
```
Comment (mattisssa) [inline extensions/pestphp-documentation/src/pestphp.tsx:26]: Is this alright to share the API key? I'm curious if there are any limits per key that end-users can easily reach.
Probably worth having it as preferences so anyone interested can configure their own tokens?

**Review by mattisssa — APPROVED** [review]
Hey, thanks for the contribution! Everything is good, I'm just not sure about one thing I mentioned below.

File: extensions/pestphp-documentation/src/pestphp.tsx (hunk 1 above)
Comment (danyelkeddah) [reply extensions/pestphp-documentation/src/pestphp.tsx:26]: i believe this is Search-only API key provided by algolia safe to use it on client side, its per site and only data owner can generate them. however pestphp documentation site are using same APIKEY.

**Comment by danyelkeddah** [comment]
Thanks @mattisssa just answered your question and i am open for any other questions or concerns.

**Pull request merged**